WindowListGroup@jake.phy@gmail.com/files/WindowListGroup@jake.phy@gmail.com/locale/hu.po

```diff
@@ -0,0 +1,340 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Can you delete this file from this PR and submit it as a separate PR? Please see [contributing](https://github.com/linuxmint/cinnamon-spices-applets/blob/master/.github/CONTRIBUTING.md), and update the PR title.